specification/e-referrals-service-api.yaml

```diff
@@ -1591,6 +1591,96 @@ paths:
           $ref: 'components/schemas/responses/TooManyRequests.yaml'
         '500':
           $ref: 'components/schemas/responses/InternalServerError.yaml'   
+  /STU3/CommunicationRequest/{ubrn}/$ers.sendCommunicationToRequester:
+    post:
+      security:
+        - bearerAuth: []
+      description: |
+        ## Overview
+        Use this endpoint, as a service provider, to respond to an advice and guidance request your service provider organisation has received. 
+        You can either provide guidance back to the referrer or request further information from the referrer. 
+        This will return the advice and guidance request back to the referrer for action.
+
+        Note:  If you want to convert an advice and guidance request into a referral, 
```

Comment:
Not really part of your changes but I think we should be careful with outputting internal terminology here. I think we should be referencing "advice and guidance requests" as `CommunicationRequests` to be using a consistent terminology throughout the xAPI.